README.md

```diff
@@ -101,7 +101,14 @@ If you need more customization, you can write a module:
 
 There are a few of types of modules:
 
-
+- Server - Use to set properties of the Bucky Server
```

Comment:
This is describing a new type of module.  Where is that module defined and used in the source?